### PR TITLE
Get breakout rooms participants

### PR DESCRIPTION
**Pull request by marcoambrosini**
Screenshot 2023-02-08 at 16 45 29

### CODE REVIEW

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue [hunk 1]
```diff
@@ -186,13 +192,35 @@ export default {
 		},
 	},
 
-	mounted() {
+	watch: {
+		isActive(newValue) {
```
Comment (nickvergessen) [inline src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:196]: So everytime you go to this tab it immediately starts a new sync of the participants and starts the 30s interval.
Should be fine enough to do it on mount only?

File: src/services/breakoutRoomsService.js [hunk 2]
```diff
@@ -95,11 +94,25 @@ const broadcastMessageToBreakoutRooms = async function(message, token) {
...
+ *
+ * @param token the conversation token
+ * @param message The message to be posted
+ * @return {Promise<AxiosResponse<any>>} The array of conversations
+ */
+const getBreakoutRoomsParticipants = async function(token) {
+	return await axios.get(generateOcsUrl('/apps/spreed/api/v4/room/{token}/breakout-rooms/participants', {
+		token,
+	})
+	)
```
Comment (nickvergessen) [inline src/services/breakoutRoomsService.js:107]:
```suggestion
	}))
```

File: src/store/breakoutRoomsStore.js [hunk 3]
```diff
@@ -146,6 +150,22 @@ const actions = {
...
+			response.data.ocs.data.forEach(participant => {
+				context.dispatch('addParticipant', {
+					token: participant.roomToken,
+					participant,
+				})
+			})
+
+		} catch (error) {
+			console.error(error)
+			showError(t('spreed', 'An error occurred while sending a message to the breakout rooms'))
```
Comment (nickvergessen) [inline src/store/breakoutRoomsStore.js:166]: wrong error message

File: src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue (hunk 1 above)
Comment (marcoambrosini) [reply src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue:196]: It's needed, but we're missing the part where we stop the interval if you navigate away from the tab, fixing this now

Comment (marcoambrosini) [reply src/components/RightSidebar/BreakoutRooms/BreakoutRoomsTab.vue]: Actually, it's in mounted that it's not needed

File: src/store/breakoutRoomsStore.js (hunk 3 above)
Comment (marcoambrosini) [reply src/store/breakoutRoomsStore.js]: Removed as it's useless for the users

**Pull request merged**